foxpuppet/windows/browser.py

```diff
@@ -10,7 +10,10 @@
 
 class BrowserWindow(BaseWindow):
 
-    """Representation of a browser window."""
+    """Representation of a browser window.
+
+        :extends: :class:`foxpuppet.windows.BaseWindow`
```

Comment:
As `BaseWindow` is not documented, let's exclude this and just allow inherited methods to be included.